go-controller/pkg/node/gateway_iptables.go

```diff
@@ -366,6 +366,40 @@ func getGatewayForwardRules(cidrs []*net.IPNet) []nodeipt.Rule {
 	return returnRules
 }
 
+func getStaleMasqueradeIptablesRules(masqueradeIP net.IP) []nodeipt.Rule {
+	var returnRules []nodeipt.Rule
```

Comment:
there is no reason to define `returnRules`, why not:
```
return []nodeipt.Rule{
		{
			Table: "filter",
			Chain: "FORWARD",
			Args: []string{
				"-s", masqueradeIP.String(),
				"-j", "ACCEPT",
			},
			Protocol: getIPTablesProtocol(masqueradeIP.String()),
		},
		{
			Table: "filter",
			Chain: "FORWARD",
			Args: []string{
				"-d", masqueradeIP.String(),
				"-j", "ACCEPT",
			},
			Protocol: getIPTablesProtocol(masqueradeIP.String()),
		},
		{
			Table: "nat",
			Chain: "POSTROUTING",
			Args: []string{
				"-s", masqueradeIP.String(),
				"-j", "MASQUERADE",
			},
			Protocol: getIPTablesProtocol(masqueradeIP.String()),
		},
	}
```

Reply:
yaa, makes sense